qadence/types.py

```diff
@@ -459,3 +461,29 @@ class ExperimentTrackingTool(StrEnum):
 
 
 LoggablePlotFunction = Callable[[Module, int], tuple[str, Figure]]
+
+
+class AnalogNoise(StrEnum):
+    """Type of noise protocol."""
+
+    DEPOLARIZING = "Depolarizing"
+    DEPHASING = "Dephasing"
+
+
+class ReadoutNoise(StrEnum):
```

Comment:
Yeah unsure we need to maintain an `StrEnum` for a single variant.

Reply:
At the moment no, but there may be other readout types implemented at a later stage.

Reply:
But I will replace it for now and do this change when we introduce another readout protocol